_pages/governance.md

```diff
@@ -176,7 +183,8 @@ Specifically out of scope would be the APIs internal to the Linux kernel. The Li
 
 #### Adding New APIs
 
-TBD
+A new API shall not break the existing one. In such case the deprecating
```

Comment:
what about?

#### Updating existing  APIs
An update of the API shall not break the existing one. When possible a new API should be created instead of updating the existing one, else the the deprecating APIs process described below shall be applied.